### DISCUSSION TITLE
Mosaic from multiple stream

### DISCUSSION THREAD

**Discussion by azrin1972** [OSNVR/OS-NVR#3] [Ideas]
Hi,

I'm wondering if we could restream multiple camera source into a single mosaic stream?

**Comment by Curid** [comment]
This could be done, but doing it server-side requires transcoding all streams and would be very demanding. Do you want to consume it with a particular client that doesn't support mosaic?

Sorry for the late reply, Github didn't notify me.

**Comment by azrin1972** [comment]
The client is a simple rtsp player. It does not support mosaic. If there’s a feature to send the mosaic as a single rtsp stream would be a great feature to haveAzrin ArisOn Nov 2, 2023, at 05:00, Curid ***@***.***> wrote:﻿
This could be done, but doing it server-side requires transcoding all streams and would be very demanding. Do you want to consume it with a particular client that doesn't support mosaic?
Sorry for the late reply, Github didn't notify me.
